File: challenges/2-tabuada/c/tabuada.c
#include <stdio.h>

int main(int argc, char const *argv[])
{
  int input;
  scanf("%d", &input);
  for (int i = 1; i <= input; i++)
  {
    printf("===========================\n");
    printf("A tabuada do numero %d e:\n", i);
    for (int j = 1; j <= 10; j++)
    {
      printf("%d * %d = %d\n", i, input, i * j);
    }
  }

  return 0;
}

File: challenges/2-tabuada/python/tabuada.py
print('Tabuada')
x = int(input('Digite um numero: '))
for c in range(1, x+1):
  print(f'A tabuada do número {c} é:')
  for n in range(1,11):
    print(f'{c}*{n} = {c*n}')